tests/unit/components/browse/ContentCardSection.spec.js

```diff
@@ -80,5 +80,39 @@ describe('components/browse/ContentCardSection', () => {
       moreButton.text().should.contain('Show more art');
       moreButton.attributes('href').should.eq('http://europeana.eu');
     });
+
+    it('displays mini cards if a section is exclusively people', () => {
```

Comment:
let's add another test where one of the entity is _not_ an agent, but a concept. Make sure       `wrapper.vm.isPeopleSection.should.equal(false);`